hack/example-app-alerting-rule.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
In our e2e tests we already use a similar alert to this, would be good to maybe use this in the tests as well so we are consistent with it?

https://github.com/openshift/cluster-monitoring-operator/blob/master/test/e2e/user_workload_monitoring_test.go#L357-L373

Reply:
Thank you Lily for your review! I have now updated this example to match the e2e test example as closely as I could.